lib/praxis/restful_doc_generator.rb

```diff
@@ -285,7 +290,8 @@ def write_index( versioned_types )
 
         reportable_types.each do |type|
           index[version] ||= Hash.new
-          display_name = type.name.split("::").last
+          next unless type.respond_to?(:display_name)
```

Comment:
I'm not sure we can "skip" if it doesn't respond to display_name can we?...we probably need get our best guess with the split.last...stuff? or can we make sure that any attr-type/blueprint-based object will respond to that?